### ISSUE TITLE
Don't sum shared currencies

### ISSUE THREAD

**Issue by SpareSimian** [Canettieri/currenciesmulti#40]
Trader's Tender is an account-wide currency. It makes no sense to display a sum of what all alts possess. There may be other currencies like this.

**Comment by Protuhj** [comment]
I can't think of other currencies that are shared, can you name any?

The NoAltCurrencyPlugin can be used for future currencies that are shared.

**Comment by Canettieri** [comment]
You are right, @Protuhj. NoAltCurrencyPlugin was made for these cases and Trader's Tender is already using it. I think we can close this issue if @SpareSimian has no more doubts.

**Comment by SpareSimian** [comment]
Thanks. I just enabled that and it looks like it's doing what we want.
